dev-lang/python2.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import os
+import stdlib
+from stdlib.template import autotools
+from stdlib.template.configure import configure
+from stdlib.split.drain_all import drain_all_with_doc
+from stdlib.manifest import manifest
+
+
+@manifest(
+    name='python2',
+    category='dev-lang',
+    description='''
+        A programming language that lets you work quickly and integrate systems more effectively.
+    ''',
+    tags=['python', 'python2', 'language', 'scripting'],
+    maintainer='dorian.trubelle@epitech.eu',
+    licenses=[stdlib.license.License.CUSTOM],
+    upstream_url='https://www.python.org/',
+    kind=stdlib.kind.Kind.EFFECTIVE,
+    versions_data=[
+        {
+            'semver': '2.7.17',
+            'fetch': [{
+                'url': 'https://www.python.org/ftp/python/2.7.17/Python-2.7.17.tar.xz',
+                'sha256': '4d43f033cdbd0aa7b7023c81b0e986fd11e653b5248dac9144d508f11812ba41',
+            },
+            ],
+        },
+    ],
+    build_dependencies=[
+        'sys-libs/zlib-dev'
+    ]
+)
+def build(build):
+    packages = autotools.build(
+        configure=lambda: configure("--with-system-expat    \
+            --with-system-ffi                               \
+            --without-ensurepip                             \
+            --enable-unicode=ucs4                           \
+            --enable-optimizations                          \
+        "),
+        split=drain_all_with_doc,
+    )
+
+    # After installing this package, it will remove pip3. This command is to reset pip and pip3.
+    stdlib.cmd("python3 -m pip install --force pip")
```

Comment:
Wtf? Why?

(`python2` should not include `pip2`. If you want `pip2`, please make another package to be consistent with `python3`)

Reply:
makes sense